brave/brave-core
# Redesign Wallet navigation and merge assets and NFTs into single section

## Resolves https://github.com/brave/brave-browser/issues/32411

### Light Mode 

<img width="713" alt="Screenshot 2023-08-22 at 6 10 01 PM" src="https://github.com/brave/brave-core/assets/3308503/78af4aa7-146b-468a-8ca2-b5c0e51c2827">

### Dark Mode

<img width="713" alt="Screenshot 2023-08-22 at 6 09 36 PM" src="https://github.com/brave/brave-core/assets/3308503/dad0c3bb-4251-44de-b705-a810d882c376">




## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Open Brave Wallet
* Observe NFTs section is not present anymore in the bottom navigation view
* Observe the new tab layout is present at the top
* Tap on NFTs tab
* Observe the NFT grid is shown
